code/game/machinery/hologram.dm

```diff
@@ -262,20 +262,24 @@ GLOBAL_LIST_EMPTY(holopads)
 	updateDialog()
 
 //do not allow AIs to answer calls or people will use it to meta the AI satellite
-/obj/machinery/hologram/holopad/attack_ai(mob/living/silicon/ai/user)
-	if(!istype(user))
+/obj/machinery/hologram/holopad/attack_ai(mob/living/silicon/ai_or_robot)
+	var/mob/living/silicon/ai/ai = ai_or_robot
+	var/mob/living/silicon/robot/robot = ai_or_robot
+	if(!istype(ai) && !istype(robot))
 		return
 	if(outgoing_call)
 		return
+	if(istype(robot))
+		interact(robot)
```

Comment:
Robots have nothing to do with what comes after, you could return here.

Reply:
It's an `if else if else` chain, it doesn't handle everything under it already